UNTITLED (1).js

```diff
@@ -0,0 +1,244 @@
+/*
+@author Dhruv
+@author Sampanna
+
+*/
+const wall = "w";
+const player = "p";
+const goal = "g";
+setLegend(
+  [ wall, bitmap`
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000
+0000000000000000`],[player, bitmap`
+................
+................
+...0000000000...
+...0000000000...
+...0222002220...
+...0022002200...
+...0000000000...
+.....002200.....
+......CCCC......
+......CCCC......
+......CCCC......
+......CCCC......
+......CCCC......
+......CCCC......
+....00000000....
+....00000000....`],  [ goal, bitmap`
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444`]
+)
+let level = 3;
```

Comment:
good new levels!
to start correctly:
`let level = 0;`

Reply:
Oh yea, I forgot to change it back to 0.